Sources/PostgresNIO/New/Messages/Close.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  File.swift
+//  
+//
+//  Created by Fabian Fett on 11.01.21.
+//
+
+extension PSQLFrontendMessage {
```

Comment:
Ditch Xcode template headers.
```suggestion
extension PSQLFrontendMessage {
```